### PR TITLE
Addding config to scripts

### PR DESCRIPTION
**Pull request by afmsavage**
This branch will be tracking my work on adding a config across scripts for easier setup/cross OS setup opportunity.

### CODE REVIEW

File: all_available_tdts.js [hunk 1]
```diff
@@ -5,24 +5,31 @@ const TBTCDepositToken = require("@keep-network/tbtc/artifacts/TBTCDepositToken.
 const Deposit = require("@keep-network/tbtc/artifacts/Deposit.json");
 const VendingMachine = require("@keep-network/tbtc/artifacts/VendingMachine.json");
 
+const config = require('./config')
+const infura = config.infura_secret || process.env.INFURA_API;
+const network = config.network || 'homestead'
```
Comment (knarz) [inline all_available_tdts.js:10]: Switching between mainnet and testnet is not as easy; the ABIs of the smart contracts might differ, so we cannot just change like that. In addition, all the `networks["1"]` would need to be changed to `networks["3"]`.

File: faucet.js [hunk 2]
```diff
@@ -2,7 +2,11 @@ const fs = require('fs');
 const https = require('https');
 const ethers = require('ethers');
 
-const ip = new ethers.providers.InfuraProvider('homestead', process.env.INFURA_API);
+const config = require('./config')
+const infura = config.infura_secret || process.env.INFURA_API;
+const network = config.network || 'homestead'
```
Comment (knarz) [inline faucet.js:7]: The faucet doesn't need infura in its current form

**Review by knarz — CHANGES_REQUESTED** [review]

File: all_available_tdts.js (hunk 1 above)
Comment (afmsavage) [reply all_available_tdts.js:10]: oh true, good catch.  I will revert that and recommit.

**Comment by afmsavage** [comment]
I made the changes.  I also could make it so that if the network was chosen as say Ropsten, it would change all the `[ 1 ]`s on the abis to be `[ 3 ]` but that might be more work than its worth.

**Comment by knarz** [comment]
Changing between mainnet and testnet might also not work easily because the smart contracts can change.
Ready to merge once you resolve the conflicts.

**Review by knarz — APPROVED** [review]

**Pull request merged**